src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -282,7 +282,7 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         if ($this->isConfigEnabled($container, $config['request'])) {
-            $this->registerRequestConfiguration($config['request'], $container, $loader);
+            $this->registerRequestConfiguration($config['request'], $container, $phpLoader);
```

Comment:
Your variable `$phpLoader`,  it's not defined

Reply:
it is defined on top

Reply:
> Your variable `$phpLoader`, it's not defined

it is defined in line 174